docs/how-to/examples.md

```diff
@@ -519,18 +519,39 @@ from ehrql.tables.core import clinical_events, patients
 hba1c_codelist = codelist_from_csv("XXX", column="YYY")
 
 dataset = create_dataset()
-max_hba1c = clinical_events.where(
+
+hba1c_events = clinical_events.where(
         clinical_events.snomedct_code.is_in(hba1c_codelist)
 ).where(
         clinical_events.date.is_on_or_after("2022-07-01")
-).numeric_value.maximum_for_patient()
+)
+
+earliest_min_hba1c_event = hba1c_events.sort_by(
+        clinical_events.numeric_value, clinical_events.date
+).first_for_patient()
+
+earliest_max_hba1c_event = hba1c_events.sort_by(
+        -clinical_events.numeric_value, clinical_events.date
```

Comment:
I reckon this is worth a comment, something like:
```suggestion
        # Note the leading minus sign to sort numeric_value in reverse order
        -clinical_events.numeric_value, clinical_events.date
```

Particularly on the assumption that this example will get copied around.